mifosng-android/src/main/java/com/mifos/mifosxdroid/online/loanaccount/LoanAccountFragment.java

```diff
@@ -352,9 +352,9 @@ public void onClick(View view) {
     public void inflatedisbusmentDate() {
         mfDatePicker = MFDatePicker.newInsance(this);
 
-        tv_disbursementon_date.setText(MFDatePicker.getDatePickedAsString());
+        tvDisbursementOnDate.setText(MFDatePicker.getDatePickedAsString());
 
-        tv_disbursementon_date.setOnClickListener(new View.OnClickListener() {
+        tvDisbursementOnDate.setOnClickListener(new View.OnClickListener() {
```

Comment:
We are using ButterKnife. Please remove this OnClickListener implementation and use Butter Knife ```@OnClick``` annotation.